pages-domain/src/main/java/nl/vpro/domain/page/RelationDefinitionService.java

```diff
@@ -14,8 +14,7 @@
     // TODO
     static RelationDefinitionService getInstance() {
         try {
-            return (RelationDefinitionService) Class.forName("nl.vpro.services.page.RelationDefinitionServiceImpl").getMethod("getInstance").invoke(null);
-        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException | ClassNotFoundException e) {
+            return (RelationDefinitionService) Class.forName("nl.vpro.domain.page.RelationDefinitionServiceImpl").getMethod("getInstance").invoke(null); } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException | ClassNotFoundException e) {
```

Comment:
Interessant. maar dat is toch ook niet het juiste class name.

Ik stel voor om het voorlopig gewoon andersom te wiren.

RelationDEfintionServiceImpl zit wel RelationDefinitionService, dus hij kan zit zelfs gewoon registreren, met 'setInstance' of zo. Dan wort het door spring in publisher geïnstantieerd, en kan het gedoe met reflections weg.

Als er meer dan 1 relationdefinitionSservice zou zijn, of geen, kunnen we op dat moment nog foutmeldingen gooien/loggen.